libs/dh/shared/feature-authorization/src/lib/permission.service.spec.ts

```diff
@@ -20,7 +20,7 @@ import { firstValueFrom, of } from 'rxjs';
 import { ActorTokenService } from './actor-token.service';
 
 describe(PermissionService.name, () => {
-  // base64 encoded access token: { role: ['actor:manage'] }
+  // base64 encoded access token: { role: ['actors:manage'] }
   const fakeAccessToken = 'ignored.eyJyb2xlIjpbImFjdG9yOm1hbmFnZSJdfQ';
```

Comment:
suggestion: the token here should probably be updated as well since it still encodes `actor:manage`

Reply:
Ye, saw that too. :D Done.